BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -179,45 +183,109 @@ public final class BlueprintView: UIView {
             )
         }
 
-        return element.content.measure(
-            in: measurementConstraint(with: size),
+        let constraint = measurementConstraint(with: fittingSize)
+
+        if let cachedSize = sizesThatFit[constraint] {
+            return cachedSize
+        }
+
+        let measurement = element.content.measure(
+            in: constraint,
             environment: makeEnvironment(),
             cache: CacheFactory.makeCache(name: "sizeThatFits:\(type(of: element))")
         )
+
+        sizesThatFit[constraint] = measurement
+
+        return measurement
+    }
+
+    ///
+    /// Measures the size needed to display the view within then given constraining size,
+    /// by measuring the current `element` of the `BlueprintView`.
+    ///
+    /// If you would like to not constrain the measurement in a given axis,
+    /// pass `0.0` or `.greatestFiniteMagnitude` for that axis.
```

Comment:
I honestly don't know how these work at all. I think as a medium, I'll make sure the `withHorizontalFittingPriority:,verticalFittingPriority:` vary based on priority.

Reply:
(Also, since these are both sizes, not just dimensions, and we don't really have "content hugging" or resistance in blueprint as a concept, I don't think it's super useful...)

Reply:
Actually, yeah, I'm not sure theres a lot for us to do here. Only the size is really useful here for us – the priorities mean nothing without a constraint system.